login.php

```diff
@@ -23,10 +23,11 @@
  *
  *  fUsername
  *  fPassword
+ *  token
  *  lang
  */
 
-define('POSTFIXADMIN_LOGOUT', 1);
+//define('POSTFIXADMIN_LOGOUT', 1);
```

Comment:
Most of your changes make sense and are a good idea, but - why do you drop the POSTFIXADMIN_LOGOUT define? It is used in common.php to log out a user whenever going to login.php, and without that define that won't happen (and you are still logged in)...